### PR TITLE
Red racoon suit

### PR DESCRIPTION
**Pull request by lzk228**
## About the PR

add red racoon suit and hat
## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

no cl no fun

### CODE REVIEW

**Comment by lzk228** [comment]
shit i writed red panda lol

**Comment by CrigCrag** [comment]
spiffo

**Comment by deltanedas** [comment]
project spessoid

**Comment by notquitehadouken** [comment]
as long as we don't say spiffo

**Comment by Kadeo64** [comment]
shit PR doesn't even add a tail attack

**Comment by Kadeo64** [comment]
(I thought this was a mario reference)

File: Resources/Prototypes/Entities/Clothing/OuterClothing/base_clothingouter.yml [hunk 1]
```diff
@@ -133,6 +133,3 @@
   components:
   - type: Item
     size: Huge
-  - type: Clothing
```
Comment (TheShuEd) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/base_clothingouter.yml:136]: why u change it?

**Comment by TheShuEd** [comment]
also, test fail is real 
image

File: Resources/Prototypes/Entities/Clothing/OuterClothing/base_clothingouter.yml (hunk 1 above)
Comment (lzk228) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/base_clothingouter.yml:136]: because slots writted in parent

**Review by TheShuEd — APPROVED** [review]

**Pull request merged**